_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,392 @@
+this past year. The company annually recognizes some of the top new open source
+projects that started during the year.
+
+### ICCF Holland Donations
+
+As mentioned in the last newsletter, [Unixstickers][iccf-unixstickers] has added
+[Neovim stickers][iccf-stickers] to their inventory. Every sticker that is
+bought increases the amount donated to [ICCF Holland][iccf-site], Bram
+Moolenaar's charity.
+
+Since they've been added, USD 106 has been raised which Unixstickers [just
+donated][iccf-tweet] on March 24th.
+
+### Related Projects
+
+The [Neovim Wiki][related-main-wiki] contains a page titled [Related
+projects][related-wiki] that contains a list of current GUI projects, API
+clients, plugins that take advantage of Neovim features, and projects with
+direct support for Neovim.
+
+Some of the projects might be experimental which means there might be bugs and
+the projects might still be in progress.
+
+Do you know of another project that can be listed? Don't be afraid to add a link
+yourself!
+
+## Development News
+
+### Terminal UI (TUI)
+
+@tarruda has merged his [pull request][tui-pr] to reimplement the terminal user
+interface using [libtermkey][tui-libtermkey] and [unibilium][tui-unibilium].
+
+Libtermkey is a library by @leonerd that "allows easy processing of keyboard
+entry from terminal-based programs." It has the ability to handle special keys,
+[UTF-8 combining][tui-combining] and more.
+
+Unibilium is a library by @mauke that interfaces with [terminfo][tui-terminfo].
+Terminfo is a library and database that makes it easy to interface with display
+terminals without worrying about device specific details. More details can be
+found on the man page for terminfo by running: [`man 5 terminfo`][tui-manpage].
+
+Although the TUI implementation is included as part of Neovim, it is
+architecturally decoupled from the editor core: it uses the same API as external
+clients and GUIs.
+
+#### Additional Keybindings
+
+As a result of using libtermkey, Neovim can now bind [meta keys][tui-meta] and
+`Ctrl-Shift` combinations both of which weren't previously possible in the
+terminal. Be sure to [read more][tui-bindings] about the updated binding
+capabilities for additional examples and possibilities.
+
+### Builtin Terminal Emulation
+
+@tarruda has finished [his pull request][term-pr] adding a new `:terminal`
+command which when ran launches a full terminal emulator. The terminal is using
+Neovim's native buffers and windows which means the user can enter [normal
+mode][term-normal] by pressing `<C-\><C-n>`.
+
+This is meant to replace and provide more capabilities than the already existing
+Vim command, `:shell`, which just launches a shell. More explanation on the
+decision of this feature can be found in [this comment][term-comment] made by
+@tarruda.
+
+The complete documentation for this feature can be found under [`:h
+nvim-terminal-emulator`][term-docs].
```

Comment:
I've used `:h` before and there doesn't seem to be any confusion but we can move to that to be more explicit.